ngx-fudis/projects/ngx-fudis/src/lib/components/form/select/common/select-icons/select-icons.component.ts

```diff
@@ -44,8 +44,8 @@ export class SelectIconsComponent implements OnChanges {
   @Input() clearButton: boolean;
 
   /**
-   * If parent'c control is disabled
-   * TODO: to be removed when Fudis is refactored only to use FormControl's disabled state
+   * If parent'c control is disabled TODO: to be removed when Fudis is refactored only to use
```

Comment:
This TODO is actually not accurate anymore and should be deleted. We already had to switch back to providing `disabled` input to all our form components since it was too strict to expect applications to disable elements only through FormControl's `disabled` state.